### PR TITLE
fix 89 added passport information prior to the

### PR DESCRIPTION
**Pull request by Grommers00**
middle ware being used on the express this should be the base of what we
use to start getting a SAML2 system going and at least start
testing out the functionality of it

### CODE REVIEW

File: src/saml2/usingPassport.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  issuer: OIDC_BASE_URI,
+  clientID: OIDC_CLIENT_ID,
+  clientSecret: OIDC_CLIENT_SECRET,
+  authorizationURL: `${OIDC_BASE_URI}/auth`,
+  userInfoURL: `${OIDC_BASE_URI}/me`,
+  tokenURL: `${OIDC_BASE_URI}/token`,
+  callbackURL: OIDC_REDIRECT_URI,
+  passReqToCallback: true,
+},
+((req, issuer, userId, profile, accessToken, refreshToken, params, cb) => {
```
Comment (humphd) [inline src/saml2/usingPassport.js]: Let's change `cb` to `callback`

File: src/saml2/usingPassport.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  console.log('issuer:', issuer);
+  console.log('userId:', userId);
+  console.log('accessToken:', accessToken);
+  console.log('refreshToken:', refreshToken);
+  console.log('params:', params);
```
[the remark this answers is not in the document]
Comment (Grommers00) [reply src/saml2/usingPassport.js]: correct - I was hoping we could use a "dummy" html site first. #91 would also be able to use these. I wanted to keep them in, just to be able to reference everything that will be passed back and forth.

File: src/saml2/usingPassport.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+OIDC_BASE_URI=https://openid-connect.onelogin.com/oidc
+OIDC_CLIENT_ID=ddf97d-xxx-xxxx-e92920
+OIDC_CLIENT_SECRET=f021f63bxxxx71f2c6a90c9b
+OIDC_REDIRECT_URI=http://localhost:3000/oauth/callback
+*/
+
+// Configure the OpenId Connect Strategy
+// with credentials obtained from OneLogin
+const {
+  OIDC_BASE_URI, OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_REDIRECT_URI,
```
[the remark this answers is not in the document]
Comment (Grommers00) [reply src/saml2/usingPassport.js]: Okay, will make the necessary changes. Thanks for the feedback as always!

**Comment by Grommers00** [comment]
@humphd @neilong31 @varshannagarajan updated - could you guys check? I had to re-create due to a computer failure a branch, and push the new information.

**Pull request closed without merging**